parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -145,7 +145,7 @@ installStandaloneContainerd() {
 
     #if there is no containerd_version input from RP, use hardcoded version
     if [[ -z ${CONTAINERD_VERSION} ]]; then
-        CONTAINERD_VERSION="1.4.12"
+        CONTAINERD_VERSION="1.4.13"
```

Comment:
In the future, we could perhaps change it to use the `stable` containerd version from manifest.cue.

Reply:
oh, I thought I had already done this? hrm

Reply:
ah it's only in cse_install. yes, that would be good. ideally we should do it in one place and throw an err if it's empty later, not default it in three places

Reply:
fine to punt

Reply:
sounds good, that refactor would make it simpler.